app/views/layouts/_application.html.erb

```diff
@@ -1,29 +1,31 @@
-    <% if Rails.env.to_s.in? Sentry.configuration.enabled_environments %>
+    <% if Rails.env.to_s.in?(Sentry.configuration.enabled_environments) && ENV['RAILS_ENV'] != 'test' %>
       <%= tag :meta, name: :release, content: Sentry.configuration.release %>
       <%= tag :meta, name: :"sentry-dsn", content: Sentry.configuration.dsn.to_s %>
       <%= tag :meta, name: :"sentry-trace", content: Sentry.get_current_scope&.span&.to_sentry_trace %>
-      <%= javascript_pack_tag('sentry') unless params[:nojs] == "nojs" %>
+      <% pack_tag_args << 'sentry' %>
     <% end %>
 
     <title><%= (yield(:title) + " - " unless yield(:title).blank?).to_s + "Manage training for early career teachers" %></title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= canonical_tag %>
     <%= tag :meta, name: 'viewport', content: 'width=device-width, initial-scale=1' %>
-    <%= tag :meta, property: 'og:image', content: asset_pack_path('media/images/govuk-opengraph-image.png') %>
+    <%= tag :meta, property: 'og:image', content: asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-opengraph-image.png') %>
     <%= tag :meta, name: 'theme-color', content: '#0b0c0c' %>
-    <%= favicon_link_tag asset_pack_path('media/images/favicon.ico') %>
-    <%= favicon_link_tag asset_pack_path('media/images/govuk-mask-icon.svg'), rel: 'mask-icon', type: 'image/svg', color: "#0b0c0c" %>
-    <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon.png'), rel: 'apple-touch-icon', type: 'image/png' %>
-    <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-152x152.png'), rel: 'apple-touch-icon', type: 'image/png', size: '152x152' %>
-    <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-167x167.png'), rel: 'apple-touch-icon', type: 'image/png', size: '167x167' %>
-    <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/favicon.ico') %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-mask-icon.svg'), rel: 'mask-icon', type: 'image/svg', color: "#0b0c0c" %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-apple-touch-icon.png'), rel: 'apple-touch-icon', type: 'image/png' %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-152x152.png'), rel: 'apple-touch-icon', type: 'image/png', size: '152x152' %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-167x167.png'), rel: 'apple-touch-icon', type: 'image/png', size: '167x167' %>
+    <%= favicon_link_tag asset_pack_path('static/govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
```

Comment:
often static assets are served by web servers instead of Rails itself.
I'm not sure whether it's our case however, if it is then I guess we should check whether this could break anything.

Reply:
That all it does is linking the static assets really. It will know how to build the path to the static assets, no matter how we serve them.